main.js

```diff
@@ -0,0 +1,20 @@
+var icon = document.getElementById("icon");
```

Comment:
Try to avoid using var as you can, its preferable to use const here instead of var